src/Salt/UserBundle/Resources/views/User/index.html.twig

```diff
@@ -2,8 +2,26 @@
 
 {% block body %}
     <h1>User list</h1>
-
-    <table class="table table-condensed table-hover table-bordered table-striped">
+    <div class="row bottomPadd8">
+        <form class="form-inline">
+            <div class="form-group col-md-4" >
+                <label for="organization">Organization : </label> 
+                <input type ="text" class="form-control" autocomplete = "off" id="search_form_organization" name="organization">
+            </div>
+            <div class="form-group col-md-4">
+                <label for="user_role">User Role : </label>
+                <select id ="search_form_user_role" class="form-control">
+                    <option value="">Select</option>
```

Comment:
We shouldn't hardcode the roles because I believe we will be adding some new roles in the short future